.github/workflows/main.yml

```diff
@@ -51,9 +52,9 @@ jobs:
           credentials_json: ${{ secrets.GCR_KEY }}
       - name: Setup Google Cloud SDK
         uses: google-github-actions/setup-gcloud@v2
-      - name: Configure GCR
-        run: |
+      - run: |
           gcloud auth configure-docker
```

Comment:
Pubsub elulator and survey are still within the GCR. Auth is needed for both projects